packages/governance/test/unitTests/test-singlePluralityBallotCount.js

```diff
@@ -0,0 +1,276 @@
+  );
+  const aliceTemplate = publicFacet.getQuestion();
+  const aliceSeat = makeHandle('Voter');
+
+  const alicePositions = aliceTemplate.getDetails().positions;
+  t.deepEqual(alicePositions.length, 3);
+  t.deepEqual(alicePositions[0], FISH);
+
+  await t.throwsAsync(
+    () => E(creatorFacet).submitVote(aliceSeat, [harden({ text: 'no' })]),
+    {
+      message: `The specified choice is not a legal position: {"text":"no"}.`,
+    },
+  );
+});
+
+test('single plurality contested', async t => {
+  const questionSpec = coerceQuestionSpec({
+    method: ChoiceMethod.PLURALITY,
+    issue: SIMPLE_ISSUE,
+    positions: [FISH, BEEF, PORK],
+    electionType: ElectionType.SURVEY,
+    maxChoices: 1,
+    maxWinners: 1,
+    closingRule: FAKE_CLOSING_RULE,
+    quorumRule: QuorumRule.MAJORITY,
+    tieOutcome: BEEF,
+  });
+  const { publisher, storageRoot } = makePublisherFromFakes();
+  const { publicFacet, creatorFacet, closeFacet } =
+    makeSinglePluralityVoteCounter(
+      questionSpec,
+      0n,
+      FAKE_COUNTER_INSTANCE,
+      publisher,
+    );
+
+  const template = publicFacet.getQuestion();
+  const aliceSeat = makeHandle('Voter');
+  const bobSeat = makeHandle('Voter');
+  const tedSeat = makeHandle('Voter');
+
+  const positions = template.getDetails().positions;
+  t.deepEqual(positions.length, 3);
+
+  E(creatorFacet).submitVote(aliceSeat, [positions[0]], 2n);
+  E(creatorFacet).submitVote(bobSeat, [positions[2]], 15n);
+  await E(creatorFacet).submitVote(tedSeat, [positions[1]], 27n);
+  closeFacet.closeVoting();
+
+  const outcome = await E(publicFacet).getOutcome();
+  t.deepEqual(outcome, BEEF);
+
+  await eventLoopIteration();
+  t.like(storageRoot.getBody('mockChainStorageRoot'), {
+    outcome: 'win',
+    position: BEEF,
+  });
+});
+
+test('single plurality tiecome', async t => {
+  const questionSpec = coerceQuestionSpec({
+    method: ChoiceMethod.PLURALITY,
+    issue: SIMPLE_ISSUE,
+    positions: [FISH, BEEF, PORK],
+    electionType: ElectionType.SURVEY,
+    maxChoices: 1,
+    maxWinners: 1,
+    closingRule: FAKE_CLOSING_RULE,
+    quorumRule: QuorumRule.MAJORITY,
+    tieOutcome: BEEF,
+  });
+  const { publisher, storageRoot } = makePublisherFromFakes();
+  const { publicFacet, creatorFacet, closeFacet } =
+    makeSinglePluralityVoteCounter(
+      questionSpec,
+      0n,
+      FAKE_COUNTER_INSTANCE,
+      publisher,
+    );
+
+  const template = publicFacet.getQuestion();
+  const aliceSeat = makeHandle('Voter');
+  const bobSeat = makeHandle('Voter');
+  const tedSeat = makeHandle('Voter');
+
+  const positions = template.getDetails().positions;
+  t.deepEqual(positions.length, 3);
+
+  E(creatorFacet).submitVote(aliceSeat, [positions[0]], 2n);
+  E(creatorFacet).submitVote(bobSeat, [positions[2]], 27n);
+  await E(creatorFacet).submitVote(tedSeat, [positions[1]], 27n);
+  closeFacet.closeVoting();
+
+  const outcome = await E(publicFacet).getOutcome();
+  t.deepEqual(outcome, [BEEF, PORK]);
```

Comment:
When you merge singlePlurality into multi, this should use the tie breaking to choose the `maxWinners = 1` winning candidate.